tikv-configuration-file.md

```diff
@@ -718,6 +718,10 @@ Configuration items related to RocksDB
 
 + WAL recovery mode
 + Optional values: `0` (`TolerateCorruptedTailRecords`), `1` (`AbsoluteConsistency`), `2` (`PointInTimeRecovery`), `3` (`SkipAnyCorruptedRecords`)
++ `0` (`TolerateCorruptedTailRecords`): tolerate incomplete record in trailing data on all logs
++ `1` (`AbsoluteConsistency`): we don't expect to find any corruption in the WAL
++ `2` (`PointInTimeRecovery`): recover to point-in-time consistency
++ `3` (`SkipAnyCorruptedRecords`): recovery after a disaster
```

Comment:
@ekexium Please help sync the changes made in https://github.com/pingcap/docs-cn/pull/6860/files#diff-5e0b63d087f45ab3370aeb3f520d8d1f637a6f031fb6b9f294b4addc227e24c5R760-R765 to here. Thanks~